### PR TITLE
Tycho: Updates Framework

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/tycho/updates/UpdatesController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import sirius.web.http.WebContext;
+import sirius.web.services.JSONStructuredOutput;
+
+@Register(classes = Controller.class)
+public class UpdatesController extends BizController {
+
+    @Part
+    private UpdateManager updateManager;
+
+@Part
```
Comment (sabieber) [inline src/main/java/sirius/biz/tycho/updates/UpdatesController.java:27]: Codeformat

File: src/main/java/sirius/biz/tycho/updates/UpdateManager.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        }
+
+        return Collections.unmodifiableList(updates);
+    }
+
+    private void fetchUpdates() {
+        try {
+            Limit limit = new Limit(0, 5);
+            List<UpdateInfo> nextUpdates = new ArrayList<>();
+            XMLCall call = XMLCall.to(new URL("https://blog.scireum.de/feed"));
```
Comment (sabieber) [inline src/main/java/sirius/biz/tycho/updates/UpdateManager.java:70]: Hardcoded?

File: src/main/java/sirius/biz/tycho/updates/UpdatesController.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+private Tagliatelle tagliatelle;
+
+    @Routed(value = "/tycho/updates", jsonCall = true)
+    public void checkForUpdates(WebContext webContext, JSONStructuredOutput output) throws CompileException {
+        tagliatelle.resolve("/kb/IKGVF.html.pasta").get().getPragma("tag");
```
Comment (sabieber) [inline src/main/java/sirius/biz/tycho/updates/UpdatesController.java:32]: Whats this?

**Review by sabieber — COMMENTED** [review]
Generell: JavaDoc

File: src/main/resources/component-biz.conf [hunk 4]
```diff
@@ -1592,6 +1593,21 @@ redis {
     }
 }
 
+# Configures the feed from which updates / news for the Tycho UI are loaded.
+tycho.updates {
+
+    # Specifies the ATOM feed to fethc news from
```
Comment (sabieber) [inline src/main/resources/component-biz.conf:1599]:
```suggestion
    # Specifies the ATOM feed to fetch news from
```

**Review by sabieber — APPROVED** [review]

**Review by idlira — APPROVED** [review]

**Pull request merged**